core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/client/withClient.ts

```diff
@@ -47,26 +52,65 @@ export function withClient() {
         {
             state: type<UVEState>()
         },
-        withState<ClientConfigState>(initialState),
+        withState<ClientConfigState>(clientState),
         withMethods((store) => {
             return {
                 setIsClientReady: (isClientReady: boolean) => {
                     patchState(store, { isClientReady });
                 },
-                setClientConfiguration: ({ query, params }: ClientRequestProps) => {
+                setCustomGraphQL: ({ query, variables }, shouldReturnFullGraphqlResponse) => {
                     patchState(store, {
-                        // Added this to avoid the client ready event to be triggered
-                        isClientReady: true,
-                        clientRequestProps: {
+                        shouldReturnFullGraphqlResponse,
+                        graphql: {
                             query,
-                            params
+                            variables
                         }
                     });
                 },
+                setGraphqlResponse: (graphqlResponse) => {
+                    patchState(store, { graphqlResponse });
+                },
                 resetClientConfiguration: () => {
-                    patchState(store, { ...initialState });
+                    patchState(store, { ...clientState });
                 }
             };
+        }),
+        withComputed((store) => {
+            return {
+                $customGraphqlResponse: computed(() => {
+                    if (!store.graphqlResponse()) {
+                        return null;
+                    }
+
+                    if (store.shouldReturnFullGraphqlResponse()) {
+                        return {
+                            ...store.graphqlResponse(),
+                            grapql: store.graphql()
+                        };
+                    }
+
+                    return store.graphqlResponse()?.page;
+                }),
+                $graphql: computed(() => {
+                    if (!store.graphql()) {
```

Comment:
Could be confusing for the difference between `$graphql` and `graphql`

Reply:
Done!